environments/main_python_env.yaml

```diff
@@ -0,0 +1,22 @@
+name: timelapse_ViT_env
+
+channels:
+- conda-forge
+
+dependencies:
+# this is restricted to Python 3.8 at this time due to other conflicts
```

Comment:
Recommend removing this, looks like it came from my CellProfiler environment haha.

Reply:
It did indeed haha, removed!